packages/flutter_tools/lib/src/template.dart

```diff
@@ -139,6 +139,8 @@ class Template {
   static const String copyTemplateExtension = '.copy.tmpl';
   static const String imageTemplateExtension = '.img.tmpl';
   static const String testTemplateExtension = '.test.tmpl';
+  // https://kotlinlang.org/docs/keyword-reference.html
+  static const List<String> reservedKotlinKeywords = <String>['when', 'in'];
```

Comment:
make this a top level const `kReservedKotlinKeywords` and document what it is used for too

Reply:
Done, thanks.